### PR TITLE
Now everything is purple to match the new logo

### PR DESCRIPTION
**Pull request by morgantaschuk**
At least there are fewer clashing colours?

image

### CODE REVIEW

File: miso-web/src/main/webapp/styles/style-full.css [hunk 1]
```diff
@@ -65,7 +65,11 @@ pre.wrap {
 }
 
 a {
-    color: #0066cc;
+    color: #404087;
+}
+
+a:hover { 
+    text-decoration: none;
```
Comment (djcooke) [inline miso-web/src/main/webapp/styles/style-full.css]: Disappearing underline on hover seems strange to me. Maybe reverse this - `a { text-decoration: none} a:hover { text-decoration: underline }`?

File: miso-web/src/main/webapp/WEB-INF/pages/editSample.jsp [hunk 2]
```diff
@@ -228,7 +228,7 @@
     </tr>
     <c:choose>
     <c:when test="${not detailedSample}">
-	  <tr style="background-color:yellow;">
+	  <tr class="highlightfield">
```
Comment (djcooke) [inline miso-web/src/main/webapp/WEB-INF/pages/editSample.jsp]: https://github.com/TGAC/miso-lims/pull/1881 removes the two changed elements on this page - the synthetic box and the QC status stripe, so these changes will likely be overwritten

Comment (morgantaschuk) [reply miso-web/src/main/webapp/WEB-INF/pages/editSample.jsp]: OK.

File: miso-web/src/main/webapp/styles/style-full.css (hunk 1 above)
Comment (morgantaschuk) [reply miso-web/src/main/webapp/styles/style-full.css]: Fixed.

**Pull request merged**